schedule/yast/maintenance/default_sle15sp3.yaml

```diff
@@ -0,0 +1,53 @@
+---
+# Default ordered sequence of steps for sle15sp3 maintenance update
```

Comment:
s/maintenance update/with maintenance updates/
to pass the ci, put file `schedule/yast/maintenance/default_sle15sp3.yaml` under `schedule/yast/sle/flows`